.github/CONTRIBUTING.md

```diff
@@ -30,24 +30,24 @@
    git remote add upstream https://github.com/faheel/Algos.git
    ```
 
-* If you cloned a while ago, get the latest changes from upstream:
+* If you cloned a while ago, get the latest changes from upstream (please note the identification of the master branch, not the reorganise branch):
```

Comment:
I think we'll remove `reorganise` soon, so these comments are unnecessary.

Reply:
Thanks for getting back to me. I have pushed updates to my branch to remove the mentions of the reorganize branch.